K2Engineering/K2Engineering/DataTypes/BarData.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
```

Comment:
Thanks Josef. Apologies for the number of rookie mistakes. I'm new to a lot of this code-writing/managing stuff. 

The actual reason for the classes is due to the solver this stuff interacts with. That had been updated and all the data that was previously bundled as generic objects was being dropped into a single hard-to-query list by the solver. By making them into custom classes this is prevented and the output data sets can be correctly managed and queried. So it was not just for readability.